release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1340,6 +1340,12 @@ For existing storage accounts created before this update, shared access keys are
 +
 link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 
+* Previously, the image registry was unable to run due to a permissions error in the certificate directory. This issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38885[*OCPBUGS-38885*])
+
+* Previously, the internal image registry did not correctly authenticate users on clusters configured with external OIDC users because the internal image registry used the `openshift specific user` API, which is not available on clusters configured with external OIDC users. Consequently, it was impossible for users to push or pull images to and from the internal image registry. With this update, the internal image registry uses the `SelfSubjectReview` API and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35335[(*OCPBUGS-35335*)]
```

Comment:
```suggestion
* Previously, the internal image registry did not correctly authenticate users on clusters configured with external OpenID Connect (OIDC) users because the internal image registry used the `openshift specific user` API. This API is not available on clusters configured with external OIDC users. Consequently, it was impossible for users to push or pull images to and from the internal image registry. With this update, the internal image registry uses the `SelfSubjectReview` API and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35335[*OCPBUGS-35335*]
```